Firestore/core/test/firebase/firestore/model/field_value_test.cc

```diff
@@ -379,6 +410,16 @@ TEST(FieldValue, Move) {
   clone = FieldValue::NullValue();
   EXPECT_EQ(FieldValue::NullValue(), clone);
 
+  const DatabaseId database_id("project", "database");
+  const FieldValue reference_value = FieldValue::ReferenceValue(
+      DocumentKey::FromPathString("abc"), &database_id);
+  clone = std::move(reference_value);
```

Comment:
Because `reference_value` is `const`, this must have performed a copy (`std::move` is just a *request* to move). Please remove `const` from `reference_value`; also, consider testing `reference_value` for emptiness after the move.

Reply:
Removed const here and other a few missed in previous PR.